### PR TITLE
adds Tutorial.md and changes the solution

### PR DESCRIPTION
**Pull request by deniznida**

### CODE REVIEW

**Comment by AnnJohn** [comment]
Assigned to @jmburges

File: TUTORIAL.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Failures:
+
+  1) LOOPS #countdown outputs '<n> SECOND(S)!' string to STDOUT for each count
+     Failure/Error: expect { countdown(10) }.to output(countdown_output).to_stdout
+     ArgumentError:
+       wrong number of arguments (1 for 0)
+```
+
+`Wrong number of arguments (1 for 0)` means our countdown method expects from us that we pass in an argument.
```
Comment (jmburges) [inline TUTORIAL.md]: "means our countdown method expects one argument (the `1` part of the error) but we passed in 0 (the `0` part of the error)". 

Important to link what you say, to how you knew that

File: TUTORIAL.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+```ruby
+def countdown(seconds_to_midnight)
+end
+```
+
+Now after passing in the seconds to midnight our method should use a `while-loop` to check if the seconds we passed in are greater than 0, as long as that condition is true, it should countdown the seconds and decrement them every time the loop runs until 0. When we hit 0 it should say `"HAPPY NEW YEAR"`.
```
Comment (jmburges) [inline TUTORIAL.md]: Hmmm this is a big jump. Do you know want to maybe give another `rspec` run and show that output? Or maybe show the tests themselves and say "ok so I know if the input is 2, my method needs to puts 2 times and then return Happy New Year. Let's first get it to print out the "N Seconds" stuff first. PAssed another test! now let's get the return right."

File: TUTORIAL.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    puts "#{seconds_to_midnight} SECOND(S)!"
+    seconds_to_midnight -= 1
+  end
+  "HAPPY NEW YEAR!"
+end
+```
+
+### `#countdown_with_sleep`
+
+This method will also accept an argument and countdown those seconds until we hit 0. The only difference will be, our method will use a ruby method called `sleep`. The `sleep(num_sec)` accepts an argument of seconds. In our `countdown_with_sleep` method we will pass in `1`.
```
Comment (jmburges) [inline TUTORIAL.md]: Same here. Explain that the readme asked for it take a bit longer, and that you then took a look at the rspec, saw the benchmark stuff and then noticed taht for a input of `5` it should sleep for `5` seconds. So we need to run the `sleep` command 5 times

**Comment by jmburges** [comment]
A few concerns. Assigned back to @deniznida

**Pull request merged**